examples/financial-research-agent/agents.ts

```diff
@@ -69,9 +69,10 @@ Focus on key numbers, events, or quotes that will be useful to a financial analy
 export const searchAgent = new Agent({
   name: 'FinancialSearchAgent',
   instructions: searchAgentPrompt,
-  model: 'gpt-4.1',
+  model: 'gpt-5.2',
   tools: [webSearchTool()],
-  modelSettings: { toolChoice: 'required' },
+  // gpt-5.2 does not support tool_choice="required"; use "auto" to allow tool use.
+  modelSettings: { toolChoice: 'auto' },
```

Comment:
Can you remove this modelSettings? The "auto" is default, so you don't need the settings.